pkgs/development/tools/misc/asls/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+  , makeWrapper
+  , fetchurl
+  , erlangR22
+}:
+
+stdenv.mkDerivation rec {
+  pname = "asls";
+  version = "0.3.0";
+
+  src = fetchurl {
+    url = "https://github.com/saulecabrera/asls/releases/download/v0.3.0/bin.tar.gz";
+    sha256 = "01d6v79zqw62pkv33km090nyn78zxr0vs3yvnw24ykysvqjyaqd3";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ erlangR22 ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./asls $out/bin/asls
+    chmod +x $out/bin/asls
+    wrapProgram $out/bin/asls --prefix PATH ":" ${erlangR22}/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "AssemblyScript Language Server";
+    homepage = https://github.com/saulecabrera/asls;
+    license = licenses.mit;
+    platforms = platforms.unix;
+    maintainers = [ maintainers.saulecabrera ];
```

A matter of taste, but I prefer:

```suggestion
    maintainers = with maintainers; [ saulecabrera ];
```